packages/unimodules-permissions-interface/src/PermissionsInterface.ts

```diff
@@ -11,4 +11,5 @@ export interface PermissionResponse {
   expires: PermissionExpiration;
   granted: boolean;
   canAskAgain: boolean;
+  scope: 'whenInUse' | 'always';
```

Comment:
What about `never`? 🤔

Reply:
`none` - but still it was missing. Added.